search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -104,8 +109,17 @@ public SearchQuery toSearchQuery(final UserTaskFilter filter) {
   @Override
   public List<String> toIndices(final UserTaskFilter filter) {
     if (isCamundaExporterEnabled) {
-      return List.of("tasklist-task-8.5.0_");
+      final String indexName =
+          String.format(
+              TaskTemplate.FULL_QUALIFIED_INDEX_NAME_PATTERN,
+              TaskTemplate.formatIndexPrefix(globalPrefix),
+              TaskTemplate.COMPONENT_NAME,
+              TaskTemplate.INDEX_NAME,
+              TaskTemplate.INDEX_VERSION);
+
+      return List.of(indexName);
     }
+    // TDB - The listview is used when the exporter is not enabled, this will be removed in 8.8
```

Comment:
Ah now I see. Hm but the Camunda Exporter will be standard right?

Reply:
In. 8.8 yes 👍 , This will be kept for 8.7

Reply:
Even for 8.7 we will not have an 8.7 importer. :)